client/src/components/QA/QAContext.jsx

```diff
@@ -11,26 +12,43 @@ export function useData() {
 }
 
 export function QADataProvider({ children }) {
-  const [qaData, setQAData] = useState(null);
+  const productId = useContext(ProductIDContext);
+  const [qData, setQData] = useState(null);
+  const [aData, setAData] = useState([]);
 
   useEffect(() => {
     axios({
       method: 'get',
       url: 'http://localhost:3000/qa/questions',
       params: {
-        // harded coded for now but will use context product ID
-        product_id: '65634',
+        product_id: productId,
       },
     })
       .then(({ data }) => {
-        setQAData(data.results);
+        setQData(data.results);
+        data.results.forEach((question) => {
+          axios({
+            method: 'get',
+            url: `http://localhost:3000/qa/questions/${question.question_id}/answers`,
```

Comment:
We can use `/qa/questions/...` as the route here without localhost. Will save us from hunting it down when we need to deploy. Non-breaking for now, though